learning_assistant/api.py

```diff
@@ -187,3 +187,14 @@ def get_course_id(course_run_id):
     course_data = get_cache_course_run_data(course_run_id, ['course'])
     course_key = course_data['course']
     return course_key
+
+
+def get_message_history(course_id, user, message_count):
+    """
+    Given a course run id (str), user (User), and message count (int), return the associated message history.
+
+    Returns a number of messages equal to the message_count value.
+    """
+    message_history = LearningAssistantMessage.objects.filter(
+        course_id=course_id, user=user).order_by('-created')[:message_count]
```

Comment:
Would we need the handle the case here if the message count exceeds the number of messages? (i.e. to prevent and index error)